install/deploy_final_non-compute_node.md

```diff
@@ -122,14 +130,14 @@ The steps in this section load hand-off data before a later procedure reboots th
     > **`NOTE`** This step will prompt for the root password of the NCNs.
 
     ```bash
-    kubernetes_rootfs="$(readlink -f /var/www/ncn-m002/rootfs)" &&
-    ceph_rootfs="$(readlink -f /var/www/ncn-s001/rootfs)" &&
     csi handoff bss-metadata \
         --data-file "${PITDATA}/configs/data.json" \
-        --kubernetes-file "${kubernetes_rootfs}" \
-        --storage-ceph-file "${ceph_rootfs}" && echo SUCCESS
+        --kubernetes-ims-image-id "$K8S_IMS_IMAGE_ID}" \
```

Comment:
I know this is merged, but isn't this missing an opening `{`? Or remove the closing `}` for consistency with the next line.